### PR TITLE
Master event ticket editor mmo

### PR DESCRIPTION
**Pull request by mmo-odoo**
Description of the issue/feature this PR addresses:
Task:https://www.odoo.com/web#id=26779&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
pad:https://pad.odoo.com/p/r.683e502045b3a16ae7d4b119f97c530f

Current behavior before PR:no more badge editor but a ticket report. 

Desired behavior after PR is merged:event ticket editor header and footer editable.when a print ticket for attedee report contain printed  Attende name,Ticket type, Date and time, Event name, location and barcode.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/event/report/event_event_templates.xml [hunk 1]
```diff
@@ -1,104 +1,94 @@
...
-                        <div class="col-xs-6 text-center" style="border-left:1px dashed black; border-top:1px dashed black; height: 148mm; text-center">
-                            <span t-field="o.event_id.badge_innerright"/>
+                    <div class="panel-footer">
+                        <div t-field="event.badge_footer">
+                            <div class="row">
+                                <div class="col-xs-offset-2 col-xs-10">
+                                    <strong> Terms and conditions </strong>
+                                    <ul>
+                                        <li>Please print this ticket in A4 page and bring with you to the event.</li>
+                                        <li>We expect participants to come 30 minutes before event.</li>
```
Comment (bst-odoo) [inline addons/event/report/event_event_templates.xml]: "the" is missing before event : <li>We expect participants to come 30 minutes before the event.</li>

File: addons/report/static/src/js/report.editor.js [hunk 2]
```diff
@@ -3,13 +3,17 @@ odoo.define('report.editor', function (require) {
...
+var core = require("web.core");
 var options = require('web_editor.snippets.options');
+var ajax = require("web.ajax");
 
 var web_base_url = $('html').attr('web-base-url');
 var trusted_host = utils.get_host_from_url(web_base_url);
 var trusted_protocol = utils.get_protocol_from_url(web_base_url);
 var trusted_origin = utils.build_origin(trusted_protocol, trusted_host);
 
+ajax.loadXML('/web/static/src/xml/base_common.xml', core.qweb);
```
Comment (tivisse) [inline addons/report/static/src/js/report.editor.js]: Why do you need to load base_common.xml on the report editor ?

It should would work without that no ?

File: addons/report/views/templates.xml [hunk 3]
```diff
@@ -79,6 +79,16 @@
                 <t t-call-assets="web_editor.summernote" t-css="false"/>
                 <t t-call-assets="web_editor.assets_editor" t-css="false"/>
                 <t t-call-assets="report.assets_editor" t-css="false"/>
+                <script>
```
Comment (tivisse) [inline addons/report/views/templates.xml]: It seems strange, why do you need to load this in the basic website.layout template ?

File: addons/event/report/event_event_templates.xml (hunk 1 above)
Comment (bst-odoo) [inline addons/event/report/event_event_templates.xml]: Don't forget my last comment ;)

File: addons/report/static/src/js/report.editor.js (hunk 2 above)
Comment (atp-odoo) [inline addons/report/static/src/js/report.editor.js]: due to this problem : https://drive.google.com/file/d/0B-Gp72uY7uAcY3I4dG1wWnhkbDQ/view?usp=drivesdk

we cannot able to load dialog when we want to add file/image.

File: addons/report/views/templates.xml (hunk 3 above)
Comment (atp-odoo) [inline addons/report/views/templates.xml]: got CSRF validation failed on path '/web_editor/attachment/add',

we cannot able to upload local image... without this.

Comment (tde-banana-odoo) [inline addons/report/views/templates.xml]: Ping @sle-odoo  : could you give your valuable input on that one Sir ?

File: addons/web/views/report_templates.xml [hunk 4]
```diff
@@ -67,6 +67,16 @@
                     <t t-call-assets="web_editor.summernote" t-css="false"/>
                     <t t-call-assets="web_editor.assets_editor" t-css="false"/>
                     <t t-call-assets="web.report_assets_editor" t-css="false"/>
+                    <script>
```
Comment (tde-banana-odoo) [inline addons/web/views/report_templates.xml]: @sle-odoo Ping ! 

Original explanation for this: atp-odoo on Mar 20 Member
got CSRF validation failed on path '/web_editor/attachment/add',

we cannot able to upload local image... without this.

**Pull request closed without merging**